src/webviews/main.ts

```diff
@@ -372,7 +370,8 @@ function updateWebview(ctx: vscode.ExtensionContext, webview: vscode.Webview, pa
         webview
     ).concat(ExtensionUtilities.getFilesAsVsCodeResources(jsPath, ['loadVsCodeApi.js'], webview))
 
-    const cssFiles = params.cssFiles ?? [isCloud9() ? 'base-cloud9.css' : 'base.css']
+    const baseCss = isCloud9() ? 'base-cloud9.css' : 'base.css'
+    const cssFiles = [baseCss, ...(params.cssFiles ?? [])]
     const loadCss = ExtensionUtilities.getFilesAsVsCodeResources(cssPath, [...cssFiles], webview)
```

Comment:
include base.css , this helps take advantage of CSS.

Most webviews should have zero or close to zero "custom css". There should be a basic style/layout and all forms should use it.

Reply:
Yeah it was only done this way because I didn't want to touch the legacy stuff just yet.